cdap-docs/admin-manual/source/installation/packages.rst

```diff
@@ -16,49 +16,75 @@ Manual Installation using Packages
 - HDP (Hortonworks Data Platform) clusters *not managed* with Apache Ambari.
 
 Cloudera Manager (CDH), Apache Ambari (HDP), and MapR distributions should be installed
 with our other :ref:`distribution instructions <installation-index>`.
 
 - As CDAP depends on HDFS, YARN, HBase, ZooKeeper, and (optionally) Hive and Spark, it must be installed
-  on cluster host(s) with full client configurations for these dependent services. 
+  on cluster host(s) with full client configurations for these dependent services.
 
 - The CDAP Master Service must be co-located on a cluster host with an HDFS client, a YARN
   client, an HBase client, and |---| optionally |---| Hive or Spark clients.
 
-- Note that these clients are redundant if you are co-locating the CDAP Master  
+- Note that these clients are redundant if you are co-locating the CDAP Master
   on a cluster host (or hosts, in the case of a deployment with high availability) with
-  actual services, such as the HDFS Namenode, the YARN resource manager, or the HBase
+  actual services, such as the HDFS NameNode, the YARN resource manager, or the HBase
   Master.
-  
-- You can download the `Hadoop client <http://hadoop.apache.org/releases.html#Download>`__ 
+
+- You can download the `Hadoop client <http://hadoop.apache.org/releases.html#Download>`__
   and `HBase client <http://www.apache.org/dyn/closer.cgi/hbase/>`__ libraries, and then
   install them on the hosts running CDAP services. No Hadoop or HBase services need be running.
 
-- All services run as the ``'cdap'`` user installed by the package manager.
+- All services run as the ``cdap`` user installed by the package manager. See `Create the "cdap" User`_ below.
 
 - If you are installing CDAP with the intention of using *replication,* see these
   instructions on :ref:`CDAP Replication <installation-replication>` *before* installing or starting CDAP.
 
 
 Preparing the Cluster
 =============================
-Please review the :ref:`Software Prerequisites <admin-manual-software-requirements>`, 
+Please review the :ref:`Software Prerequisites <admin-manual-software-requirements>`,
 as a configured Hadoop, HBase, and Hive (plus an optional Spark client) needs to be configured on the
-node(s) where CDAP will run. 
+node(s) where CDAP will run.
 
 .. Hadoop Configuration
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+Create the "cdap" User
+----------------------
+
+.. highlight:: console
+
+The base CDAP package will install ``cdap`` as a system user. However, in order to have a
+consistent UID for the ``cdap`` user across the cluster, or to ensure that it is created
+on the namenode when installing CDAP elsewhere, you need to check that the ``cdap`` user
+is installed |---| and installed consistently |---| on all nodes of the cluster.
+Though the base CDAP package will automatically install the ``cdap`` system user, it's
```

Comment:
it seems like this sentence is redundant now to above and below... but i'll leave it up to you